datapackage_pipelines_mojp/clearmash/api.py

```diff
@@ -71,6 +71,9 @@ def get_web_document_system_folder(self, folder_id):
     def get_documents(self, entity_ids):
         return self._wcm_api_call("/Documents/Get", {"entitiesIds": entity_ids})
 
+    def get_document_photos_docs(self, entity_id):
+        return self._wcm_api_call("/Document/ByRelationField", {"EntityId": entity_id, "FieldId": "_c6_beit_hatfutsot_bh_base_template_multimedia_photos", "MaxNestingDepth": 1})
```

Comment:
I think it would be better that the API only provides an interface over Clearmash without additional logic

so, for example, a better function would be

def get_document_by_relation_field(self, entity_id, field_id, max_nesting):

this way it will be useful for other fields as well, and it makes less assumptions about how it will be used